src/Symfony/Component/Runtime/BaseRuntime.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Runtime;
+
+/**
+ * A runtime to do bare-metal PHP without using superglobals.
+ *
+ * One option named "debug" is supported; it toggles displaying errors.
+ *
+ * The app-closure returned by the entry script must return either:
+ * - "string" to echo the response content, or
+ * - "int" to set the exit status code.
+ *
+ * The app-closure can declare arguments among either:
+ * - "array $context" to get a local array similar to $_SERVER;
+ * - "array $argv" to get the command line arguments when running on the CLI;
+ * - "array $request" to get a local array with keys "query", "data", "files" and
+ *   "session", which map to $_GET, $_POST, $FILES and &$_SESSION respectively.
+ *
+ * The runtime sets up a strict error handler that throws
+ * exceptions when a PHP warning/notice is raised.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class BaseRuntime implements RuntimeInterface
+{
+    private $debug;
+
+    public function __construct(array $options = ['debug' => true])
```

Comment:
Just a detail but I think the default `debug` value should be set only once inside the method (already done on the next following line). So this should be simplified to `$options = []`.
And it will be consistent with `SymfonyBase` and his default `env` value only set [inside the method](https://github.com/symfony/symfony/pull/36652/files#diff-18578a7bf9e590e1557d876ecf806063R74) 

Defining a default value in different places may be source of mistakes if you decide to update it in the future and forget to update one of them.

Reply:
works for me, updated.